client/src/pages/admin/mentor-registry.tsx

```diff
@@ -79,172 +98,118 @@ function Page(props: Props) {
     [modalData],
```

Comment:
I'd propose to add `loadData` and `openNotificationWithIcon`  deps array or remove `useCallback` here

Reply:
fixed